src/Common/utils.ts

```diff
@@ -154,6 +155,17 @@ class Utils {
     return this.toHex(hash)
   }
 
+  /**
+   * Check if the given string is valid hex.
+   *
+   * @param input - The input to check.
+   * @returns true if the input is valid hex, otherwise false.
+   */
+  public static isHex(input: string): boolean {
+    const hexRegEx = /(?:[0-9]|[a-f])/gimu
+    return (input.match(hexRegEx) ?? []).length === input.length
+  }
```

Comment:
We have a rule that enforces `member-ordering` in classes, and [right now it's configured](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/member-ordering.md#default-configuration) to put `public static` members before `private static` members.

I'm open to changing the ordering of class members if we have a different preference. Also, this rule allows enforcing an alphabetical sort order, which I currently have disabled, but we might want to enable.